src/network.zig

```diff
@@ -1203,44 +1180,482 @@ pub const Protocols = struct {
 			writer.writeEnum(items.Inventory.Command.PayloadType, payloadType);
 			std.debug.assert(writer.data.items[0] != 0xff);
 			writer.writeSlice(_data);
-			conn.sendImportant(id, writer.data.items);
+			conn.send(.fast, id, writer.data.items);
 		}
 		pub fn sendConfirmation(conn: *Connection, _data: []const u8) void {
 			std.debug.assert(conn.user != null);
 			var writer = utils.BinaryWriter.initCapacity(main.stackAllocator, _data.len + 1);
 			defer writer.deinit();
 			writer.writeInt(u8, 0xff);
 			writer.writeSlice(_data);
-			conn.sendImportant(id, writer.data.items);
+			conn.send(.fast, id, writer.data.items);
 		}
 		pub fn sendFailure(conn: *Connection) void {
 			std.debug.assert(conn.user != null);
-			conn.sendImportant(id, &.{0xfe});
+			conn.send(.fast, id, &.{0xfe});
 		}
 		pub fn sendSyncOperation(conn: *Connection, _data: []const u8) void {
 			std.debug.assert(conn.user != null);
 			var writer = utils.BinaryWriter.initCapacity(main.stackAllocator, _data.len + 1);
 			defer writer.deinit();
 			writer.writeInt(u8, 0);
 			writer.writeSlice(_data);
-			conn.sendImportant(id, writer.data.items);
+			conn.send(.fast, id, writer.data.items);
 		}
 	};
 };
 
 pub const Connection = struct { // MARK: Connection
-	const maxPacketSize: u32 = 65507; // max udp packet size
+	const maxMtu: u32 = 65507; // max udp packet size
 	const importantHeaderSize: u32 = 5;
-	const maxImportantPacketSize: u32 = 576 - 20 - 8; // IPv4 MTU minus IP header minus udp header
+	const minMtu: u32 = 576 - 20 - 8; // IPv4 MTU minus IP header minus udp header
```

Comment:
Although the name of the variable is alright, I'd like to point out that it literally means `minimal maximum transmission unit` which is just a bit funny. Since this is an abbreviation it should technically be all caps tho.

Reply:
No, I'm using Zig's naming convention for abbreviations, which treats them as one word.